classes/gateways/class.pmprogateway_stripe.php

```diff
@@ -3396,4 +3422,20 @@ private function convert_price_to_unit_amount( $price ) {
 
 		return intval( $price * $currency_unit_multiplier );
 	}
+	/**
+	 * Determine whether the calling method was called from
+	 * inside or outside of the Stripe Gateway class.
+	 *
+	 * Useful for preparing to change method visibility from
+	 * public to private.
+	 *
+	 * @return bool
+	 */
+	static private function method_being_called_from_outside_class() {
```

Comment:
In general this is the normal order used everywhere in PHP land and it's best to keep things familiar for devs

```suggestion
	private static function method_being_called_from_outside_class() {
```